definition/data/sheets/CaseField.json

```diff
@@ -70,10 +70,8 @@
   {"CaseTypeID": "MoneyClaimCase", "ID": "eyewitnessUploadDeadline", "Label": "Send witness statements before 4 PM", "FieldType": "Date", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "expertReportPermissionPartyAskedByDefendant", "Label": "Permission to use an expert requested by Defendant", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "expertReportPermissionPartyAskedByClaimant", "Label": "Permission to use an expert requested by Claimant", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "expertReportPermissionPartyGivenToDefendant", "Label": "Grant permission for expert?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "expertReportPermissionPartyGivenToClaimant", "Label": "Grant permission for expert?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "expertReportInstructionClaimant", "Label": "Additional instructions for expert report", "FieldType": "Collection", "FieldTypeParameter": "Text","SecurityClassification": "Public"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "expertReportInstructionDefendant", "Label": "Additional instructions for expert report", "FieldType": "Collection", "FieldTypeParameter": "Text","SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "grantExpertReportPermission", "Label": "Grant permission for expert?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
```

Comment:
please make sure changing the caseField will have impact on cases with the caseField already will start failing. Even if code goes first or definition goes first, both cases it will fail.